transcoding/README.md

```diff
@@ -18,32 +19,11 @@ This contains resources for building an application that broadcast media from an
     node ./deploy.js -b broadcast-lambda-cf-deploy-bucket -s broadcast-lambda-cf-stack -n broadcast-lambda -r us-east-1
     ```
 
-3. Once the deployment completes, it shows the ARN of Lambda function created. Go to AWS console and find the function. 
+3. Once the deployment completes, it shows the ARN of Lambda function created. Go to AWS console and find the function.
+ The arn of the broadcast lambda function will look something like this: `arn:aws:lambda:us-east-1:000087643245:function:broadcast-lambda-cf-stack-lambda`. This function will be used in the next section to start/stop the ECS task.
     
-   #### Options to start/stop transcoding
-   1. You can **START** a broadcast by creating a test event with the input below, then click **Test** button: 
-    ```
-    {
-      "action": "start",
-      "meetingURL": "<your-meeting-url-to-be-broadcast>",
-      "rtmpEndpoint": "<your-rtmp-endpoint-url>"
-    }
-    ```
-    The output of the above command is an ECS task ID, it will be used to **STOP** the broadcast. To stop the broadcast, create another test event with the input below, then click **Test** button:
-    ```
-    {
-      "action": "stop",
-      "taskId": "<your-ecs-task-id-or-arn>"
-    }
-    ```
-    2.  Use AWS client to invoke lambda to start and stop transcoding. 
-      ```
-      // Start
-      aws lambda invoke --function-name broadcast-lambda --payload '{"action":"start","meetingURL":"<your-meeting-url-to-be-broadcast>","rtmpEndpoint":"<your-rtmp-endpoint-url>"}' --cli-binary-format raw-in-base64-out output.json
-
-      // Stop  
-      aws lambda invoke --function-name broadcast-lambda --payload '{"action":"stop", "taskId": "<your-ecs-task-id-or-arn>"}' --cli-binary-format raw-in-base64-out output.json
-      ```
+## Running the transcoding task in ECS
+Please refer to the [running transcoding for broadcast](../README.md/#running-transcoding-for-broadcast) section in the main README.
 
 ## What's in Behind
```

Comment:
Whats in Behind ?  Can we modify this?

Reply:
It is the explanation of how the `transcoding` deploy script works.

Reply:
I changed the title of the section to match backend doc and added a short intro attached below.

> ## What will running deploy.js do?
> This section dives into the implementation of the deploy script. Following are the several resources created by the script.